src/civic_canister_backend/src/credential.rs

```diff
@@ -313,20 +295,20 @@ async fn remove_credential(
                     credentials.insert(principal, CredentialList(existing_credentials_vec));
                     Ok("Credential removed successfully".to_string())
                 } else {
-                    Err(CredentialError::UnauthorizedSubject(
+                    Err(IssueCredentialError::UnauthorizedSubject(
                         "Unauthorized: You do not have permission to remove this credential."
                             .to_string(),
                     ))
                 }
             } else {
-                Err(CredentialError::NoCredentialFound(format!(
+                Err(IssueCredentialError::UnauthorizedSubject(format!(
                     "Credential not found with id {} for principal {}",
                     credential_id,
                     principal.to_text()
                 )))
             }
         } else {
-            Err(CredentialError::NoCredentialFound(format!(
+            Err(IssueCredentialError::UnauthorizedSubject(format!(
```

Comment:
Here and above, UnauthorizedSubject seems wrong

Reply:
Updated